tests/test_main.py

```diff
@@ -79,4 +79,4 @@ def test_resize_bucket():
 
     for key, value in items:
         assert dictionary[key] == value
-    assert len(dictionary) == len(items)
+    assert len(dictionary) == len(items)
```

Comment:
Why did you remove empty line at the end of the file.

Reply:
When my tests gave different results with the same code, I tried to copy test code from a later version of the task if I had test code with bag (it didn't help :() and maybe I didn't copy carefully